test/verify/check-networkmanager-firewall

```diff
@@ -409,6 +409,19 @@ class TestFirewall(NetworkCase):
         # should have been removed in the reload
         b.wait_not_present("tr[data-row-id='http']")
 
+        b.click("#add-services-dialog button.btn-cancel")
+        b.click("tr[data-row-id='custom--80-82-echo-83-123-snmp'] button[aria-label=Details]")
+        b.click("tr[data-row-id='custom--80-82-echo-83-123-snmp'] button[aria-label=Actions]")
+        b.click("a:contains(Edit)")
+        b.wait_val("#tcp-ports", "80-82, 7")
+        b.wait_val("#udp-ports", "83-123, 161")
+        b.wait_val("#service-name", "custom--80-82-echo-83-123-snmp")
+        b.wait_val("#service-description", "custom snmp firewall rule")
+        b.set_input_text("#tcp-ports", "80-82, 8")
+        b.set_input_text("#service-description", "edited snmp firewall rule")
+        b.wait_val("#service-name", "custom--80-82-echo-83-123-snmp")
+        save("custom--80-82-echo-83-123-snmp", "80-82, 8", "83-123, 161", "edited snmp firewall rule", 'Edit service')
```

Comment:
Does that validate the actual settings in firewalld anywhere? That tcp/7 is really closed, tcp/8 got opened, and UDP survived?